app/views/doctors/show.html.erb

```diff
@@ -0,0 +1,33 @@
+<div class="row">
+  <div class="col-md-6 offset-md-3 text-center colorlib-heading">
+    <h2 id="patient-information">Doctor Information</h2>
+  </div>
+</div>
+<section class="home_banner_area">
+  <div class="container box_1620">
+    <div class="banner_inner d-flex align-items-center">
+      <div class="banner_content">
+        <div class="media">
+          <div class="d-flex">
+            <%= image_tag @doctor.image %>
```

Comment:
docter k upload ảnh có dẹo k e
k có ảnh có phá giao diện k

Reply:
không ạ, e toàn dùng bootstrap thôi :)))

Reply:
thử docter k upload image xem chết trang k e

Reply:
không bị sao cả chị ơi